release/src/release-log.ts

```diff
@@ -21,9 +21,13 @@ export async function gitLog(majorVersion: number) {
 
 function processCommit(commitLine: string): CommitInfo {
   const [refs, message, hash, date] = commitLine.split('||');
-  const version = refs?.match(/(v[\d\.\-RCrc]+)/)?.[1] ?? '';
+  const tags = refs?.match(/tag: ([\w\d-_\.]+)/g) ?? '';
 
-  return { version, message, hash, date};
+  const versions = tags
+    ? tags?.map((v) => v.replace('tag: ', ''))
```

Comment:
Why do we still need this conditional if we've checked `tags` is truthy in the ternion expression?

```suggestion
    ? tags.map((v) => v.replace('tag: ', ''))
```